.github/workflows/python-publish.yml

```diff
@@ -5,11 +5,26 @@ name: Upload Python Package
 
 on:
   workflow_dispatch:
-  release:
-    types: [published]
```

Comment:
Don't we need to separate workflow files? just in case.

Reply:
I think that it's not possible to separate workflow into files.

Reply:
OK